### DISCUSSION TITLE
MDC Syntax for Array of Objects?

### DISCUSSION THREAD

**Discussion by justindasilva** [nuxt/content#1478] [General]
I really like the `Releases` component that is part of Docus. I've wanted to make use of it in my application, but we don't use GitHub over here. I copied the component over to my codebase and renamed it `ChangeLog.vue` and made a few changes.

I'm hoping someone can let me know how it might be possible to pass an array of objects to a component using Nuxt Content MDC?

For example, I'm currently doing the following:
```
::change-log
---
releases: [
{
    name: v0.0.2,
    date: 2022-08-20,
    text: This is what was worked on in v0.0.2
},
{
    name: v0.0.1,
    date: 2022-08-01,
    text: This is what was worked on in v0.0.1
}
]
---
::
```

My ChangeLog component defines this prop:
```
const props = defineProps({
  releases: {
    type: Array as PropType<any>,
    default: () => [],
  },
})
```

What I see is this:
![Screen Shot 2022-08-26 at 11 06 54 AM](https://user-images.githubusercontent.com/882534/186936052-8c7586dd-154f-4008-a634-ed0f23ad5f7a.png)

This is fine, but I would really like to make use of Markdown syntax for what I supply as the `text` parameter. Is there anyway to accomplish something close to the following:

```
::change-log
---
releases:
---
name: v0.0.2,
date: 2022-08-20,
This is what was worked on in *v0.0.2*
---
name: v0.0.1,
date: 2022-08-01,
This is what was worked on in *v0.0.1*
---
::
```

**Comment by nizasichi** [comment]
Have you tried using [Slots](https://content.nuxt.com/usage/markdown#slots)?
